roles/certificate-acl/templates/certificate_acl_service.yaml.j2

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    name: certificate-acl
+  name: certificate-acl
+  namespace: {{ dojot_namespace }}
+spec:
+  ports:
+  - port: {{ dojot_certificate_acl_port }}
```

Comment:
This is not being used now, but with the changes that are being introduced at https://github.com/dojot/dojot/pull/2136 and  https://github.com/dojot/dojot/pull/2135  you will probably need the file, maybe with a different port just